doc/Language/grammar_tutorial.pod6

```diff
@@ -27,41 +27,75 @@ language.
 
 =SUBTITLE The Broad Concept of Perl Grammars
 
-Regular Expressions (regex) work well to find patterns in strings and manipulate them. However, when you need to find multiple patterns at once, or need to combine patterns, or test for patterns that may surround strings, or other patterns - regular expressions alone are not adequate.
```

Comment:
How did you do such a wrapping? I may be wrong, but, for example, my emacs gives me a different result.
For this line emacs "fill-region" does
```
Regular Expressions (regex) work well to find patterns in strings and
manipulate them. However, when you need to find multiple patterns at
once, or need to combine patterns, or test for patterns that may
surround strings, or other patterns - regular expressions alone are
not adequate.
```
but your result is
```
Regular Expressions (regex) work well to find patterns in strings and manipulate
them. However, when you need to find multiple patterns at once, or need to
combine patterns, or test for patterns that may surround strings, or other
patterns - regular expressions alone are not adequate.
```